frontend/src/utils/payment-params.ts

```diff
@@ -26,7 +26,7 @@ export class PaymentParams {
     readonly amount?: number,
     readonly expiration?: Date,
     readonly redirectUrl?: string,
-    readonly demo?: boolean,
+    readonly demo?: boolean
```

Comment:
as we agreed - demo is not optional but have false as default value

Reply:
The reason I made it optional in this file, is because if you check below on the same file, there seem to be two cases where params.length is equal to 2 or 3 (lines 39 and 63 respectively). In those cases, demo is passed as undefined, so the interface declaration needs to be optional else TS is complaining.  Does it make sense?